python/ql/test/modelling/FrameworkModeEndpoints.ext.yml

```diff
@@ -0,0 +1,20 @@
+extensions:
+  - addsTo:
+      pack: codeql/python-all
+      extensible: sourceModel
+    data:
+      - ["MyPackage.Foo.C1","Member[m1].ReturnValue","remote"]
+      - ["MyPackage","Member[Foo].Member[C1].Instance.Member[m2].ReturnValue","remote"]
+
+  - addsTo:
+      pack: codeql/python-all
+      extensible: summaryModel
+    data:
+      - ["MyPackage.Foo.C1!","Member[m3]","Argument[0]","ReturnValue","value"]
+      - ["MyPackage","Member[Foo].Member[C1].Member[m4]","Argument[0]","ReturnValue","value"]
+
+  - addsTo:
+      pack: codeql/python-all
+      extensible: typeModel
```

Comment:
Why do we have typeModels here? I thought type-models were not support by the model editor yet?

Reply:
The plan is for the UI to support type models.

Reply:
ok. I think it could have been clearer that this serves to support implementing the UI feature in the future, but now that I know, I'm not going to be insistent on it :shrug: